storybook/pages/ProfileDialogViewPage.qml

```diff
@@ -74,8 +75,8 @@ SplitView {
                                                          text: "__twitter",
                                                          url: "https://twitter.com/ethstatus",
                                                          icon: "twitter"
-                                                    },
-                                                    {
+                                                     },
```

Comment:
ehh, when we get rid of that global stuff...

Reply:
What do you mean?

Reply:
nvm, I'd like to refactor it at some point finally :)